gigaword-indexer/src/main/java/edu/isi/vista/gigawordIndexer/GigawordIndexer.java

```diff
@@ -0,0 +1,214 @@
+import de.tudarmstadt.ukp.clarin.webanno.automation.service.export.AutomationTrainingDocumentExporter;
+import de.tudarmstadt.ukp.clarin.webanno.model.Project;
+import de.tudarmstadt.ukp.clarin.webanno.model.SourceDocument;
+import de.tudarmstadt.ukp.clarin.webanno.security.UserDao;
+import de.tudarmstadt.ukp.clarin.webanno.security.model.User;
+import de.tudarmstadt.ukp.clarin.webanno.ui.annotation.AnnotationPageMenuItem;
+import de.tudarmstadt.ukp.clarin.webanno.ui.curation.page.CurationPageMenuItem;
+import de.tudarmstadt.ukp.clarin.webanno.ui.monitoring.page.AgreementPageMenuItem;
+import de.tudarmstadt.ukp.clarin.webanno.ui.monitoring.page.MonitoringPageMenuItem;
+
+@SpringBootApplication
+@ComponentScan(
+		basePackages={"de.tudarmstadt.ukp.inception.app",
+		"de.tudarmstadt.ukp.clarin.webanno"},
+		excludeFilters = {
+	        @Filter(type = FilterType.CUSTOM, classes = TypeExcludeFilter.class),
+	        @Filter(type = FilterType.CUSTOM, classes = AutoConfigurationExcludeFilter.class),
+	        @Filter(type = FilterType.ASSIGNABLE_TYPE, classes = { 
+	                // The INCEpTION dashboard uses a per-project view while WebAnno uses a global
+	                // activation strategies for menu items. Thus, we need to re-implement the menu
+	                // items for INCEpTION.
+	                AnnotationPageMenuItem.class,
+	                CurationPageMenuItem.class,
+	                MonitoringPageMenuItem.class,
+	                AgreementPageMenuItem.class,
+	                // INCEpTION uses its recommenders, not the WebAnno automation code
+	                AutomationService.class, 
+	                AutomationMiraTemplateExporter.class,
+	                AutomationTrainingDocumentExporter.class
+        })}
+)
+@EntityScan(basePackages = {
+        // Include WebAnno entity packages separately so we can skip the automation entities!
+        "de.tudarmstadt.ukp.clarin.webanno.model",
+        "de.tudarmstadt.ukp.clarin.webanno.security"})
+@ImportResource({ 
+    "classpath:application-context.xml"
+})
+public class GigawordIndexer implements CommandLineRunner{
+
+	@Autowired 
+	@Qualifier("authenticationManager1")
+	private AuthenticationManager authManager;
+
+	private @Autowired UserDao userRepository;
+    
+	private @Autowired ProjectService projectService;
+    
+	private @Autowired DocumentService documentService;
+    
+	private @Autowired AnnotationSchemaService annotationSchemaService;
+
+    private static final String PROJECT_NAME_GW = "Gigawords";
+
+    public GigawordIndexer() {
+    	
+    }
+
+    public void run(String... args) {
+    	
+    	System.out.println("GigawordIndexer is running...");
+        
+    	// TODO
+    	// parse gigaword
+        // create Project
+        // create SourceDocuments
+        // import document text
+    	
+  
+    	// see a list of users
+    	List<User> users = userRepository.list();
```

Comment:
This looks like debugging code which can get removed?

Reply:
Most of the code in this file right now is testing to see how everything works. It will be replaced by the next pull request.